app/src/aosp/java/com/igalia/wolvic/PlatformActivity.java

```diff
@@ -13,6 +13,14 @@
 
 public class PlatformActivity extends GameActivity {
 
+    public interface TrayDelegate {
```

Comment:
We should definitely not define this on every PlatformActivity class. I'd suggest having it in the traywidget for example or in another better place